docs/docs/concepts/how-it-works/_category_.json

```diff
@@ -0,0 +1,5 @@
+{
+  "position": 100,
+  "label": "How it works",
+  "collapsible": true
```

Comment:
Would it be an idea to show the section uncollapsed? I feel that this is a bit nicer in the sidebar.

```suggestion
  "collapsible": true,
  "collapsed": false
```